scripts/generate-ssl-cert.sh

```diff
@@ -2,8 +2,9 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/..";
 
-[ -f "${DIR}/opt/letsencrypt/settings"  ] && source "${DIR}/opt/letsencrypt/settings" || { echo "No settings available" ; exit 3 ; }
-[ -f "${DIR}/root/validation.sh"  ] && source "${DIR}/root/validation.sh" || { echo "No validation library available" ; exit 3 ; }
+[ -f "${DIR}/opt/letsencrypt/settings" ] && source "${DIR}/opt/letsencrypt/settings" || { echo "No settings available" ; exit 3 ; }
+[ -f "${DIR}/root/validation.sh" ] && source "${DIR}/root/validation.sh" || { echo "No validation library available" ; exit 3 ; }
+[ -f "${DIR}/var/lib/jelastic/keys/letsencrypt/settings-custom" ] && source "${DIR}/var/lib/jelastic/keys/letsencrypt/settings-custom"
```

Comment:
avoid *source*

Reply:
removed. No needs to source there, config file is written in ssl-manager script